functional-test/src/test/resources/project-import/ansible-large-inventory/rundeck-ansible-large-inventory/files/etc/project.properties

```diff
@@ -0,0 +1,31 @@
+#edit below
+project.disable.executions=false
+project.disable.schedule=false
+project.execution.history.cleanup.batch=500
+project.execution.history.cleanup.enabled=false
+project.execution.history.cleanup.retention.days=60
+project.execution.history.cleanup.retention.minimum=50
+project.execution.history.cleanup.schedule=0 0 0 1/1 * ? *
+project.jobs.gui.groupExpandLevel=1
+project.later.executions.disable.value=0
+project.later.executions.disable=false
+project.later.executions.enable.value=
+project.later.executions.enable=false
+project.later.schedule.disable.value=
+project.later.schedule.disable=false
+project.later.schedule.enable.value=
+project.later.schedule.enable=false
+project.name=ansible-large-inventory
+project.nodeCache.enabled=true
```

Comment:
perhaps you could change this to false and remove the method "waitForNodes". It SHOULD not use the nodes cache so the retrieved node list should have all the nodes